diff --git a/frontend/client/components/Proposal/CampaignBlock/index.tsx b/frontend/client/components/Proposal/CampaignBlock/index.tsx
index cf1ec18..c30eff3 100644
--- a/frontend/client/components/Proposal/CampaignBlock/index.tsx
+++ b/frontend/client/components/Proposal/CampaignBlock/index.tsx
@@ -91,26 +91,30 @@ export class ProposalCampaignBlock extends React.Component<Props, State> {
           <div className="ProposalCampaignBlock-info">
             <div className="ProposalCampaignBlock-info-label">Funding</div>
             <div className="ProposalCampaignBlock-info-value">
-              {proposal.contributionMatching > 0 && (
-                <Popover
-                  overlayClassName="ProposalCampaignBlock-popover-overlay"
-                  placement="left"
-                  content={
-                    <>
-                      <b>Matching</b>
-                      <br />
-                      Double your impact! Contributions to this proposal are being
-                      matched.
-                    </>
-                  }
-                >
-                  <span className="ProposalCampaignBlock-info-value-matching">x2 </span>
-                </Popover>
-              )}
               <UnitDisplay value={funded} /> / <UnitDisplay value={target} symbol="ZEC" />
             </div>
           </div>
 
+          {proposal.contributionMatching > 0 && (
+            <div className="ProposalCampaignBlock-matching">
+              <span>Funds are being matched x{proposal.contributionMatching + 1}</span>
+              <Popover
+                overlayClassName="ProposalCampaignBlock-popover-overlay"
+                placement="left"
+                content={
+                  <>
+                    <b>Matching</b>
+                    <br />
+                    Increase your impact! Contributions to this proposal are being
+                    matched by the Zcash Foundation, up to the target amount.
+                  </>
+                }
+              >
+                <Icon type="question-circle" theme="filled" />
+              </Popover>
+            </div>
+          )}
+
           {isFundingOver ? (
             <div
               className={classnames({
diff --git a/frontend/client/components/Proposal/CampaignBlock/style.less b/frontend/client/components/Proposal/CampaignBlock/style.less
index 2a9ff07..847086e 100644
--- a/frontend/client/components/Proposal/CampaignBlock/style.less
+++ b/frontend/client/components/Proposal/CampaignBlock/style.less
@@ -32,6 +32,19 @@
     }
   }
 
+  &-matching {
+    margin: 0.5rem -1.5rem;
+    padding: 0.75rem 1.5rem;
+    text-align: center;
+    background: @info-color;
+    color: #FFF;
+    font-size: 1rem;
+
+    .anticon {
+      margin-left: 0.5rem;
+    }
+  }
+
   &-popover {
     &-overlay {
       max-width: 400px;
